frontend/src/components/worksheets/items/TableItem.js

```diff
@@ -141,14 +151,21 @@ class TableRow extends React.Component {
         var baseUrl = this.props.url;
         var uuid = this.props.uuid;
         var columnWithHyperlinks = this.props.columnWithHyperlinks;
+        var worksheetName = this.props.worksheetName;
+        var worksheetUrl = this.props.worksheetUrl;
         var rowCells = this.props.headerItems.map(function(headerKey, col) {
             var rowContent = rowItems[headerKey];
 
             // See if there's a link
             var url;
-            if (col == 0) {
+            if (headerKey === 'host_worksheet' && worksheetUrl !== undefined) {
```

Comment:
Would it be better to check if worksheetUrl evaluates to true ( && worksheetUrl), so it handles null/''/undefined?

Reply:
I will leave it as is. In this situation, `worksheetUrl` can only be `undefined`. 

In general, we should avoid falsies (`&& worksheetUrl`). This is not idiomatic in JavaScript and in the lower level, it starts a chain of casting types, since in isolation it suggests `worksheetUrl` is a boolean, which in this case it is not.